docs/traffic-policy/secrets.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Traffic Policy Secrets
+sidebar_label: Secrets
+description: Store sensitive data in encrypted vaults and reference them dynamically in your traffic policies.
+---
+
+Secrets enable you to store sensitive data in encrypted vaults and reference them dynamically in your traffic policies.
+This feature eliminates the need to hardcode passwords, API keys, and other sensitive values directly in policy YAML files.
+
+:::info
+Secrets and vaults are currently in Developer Preview.
```

Comment:
We may be able to document this as 'open beta' (thus eliminating the 'request to join early access' requirement).

I've asked for confirmation we can eliminate this requirement here:
https://ngrok.slack.com/archives/C01CYSSHVFS/p1754513489994749